tests/unit/test_utils.py

```diff
@@ -701,6 +704,31 @@ def test_manylinux_check_glibc_version(self):
                     # Didn't find the warning we were expecting
                     assert False
 
+    def test_glibc_version_string_confstr_fail(self, monkeypatch):
+
+        def raises(error):
+            raise error()
+
+        monkeypatch.setattr(
+            os, "confstr", lambda x: raises(ValueError), raising=False,
+        )
+        assert glibc_version_string_confstr() is None
+
+        monkeypatch.setattr(
+            os, "confstr", lambda x: raises(OSError), raising=False,
+        )
+        assert glibc_version_string_confstr() is None
+
+        monkeypatch.setattr(os, "confstr", lambda x: "XXX", raising=False)
+        assert glibc_version_string_confstr() is None
```

Comment:
I think these consecutive assertions would be better done with `@pytest.mark.parametrize`.